components/dashboard/src/settings/Preferences.tsx

```diff
@@ -213,10 +198,14 @@ function orderedIdeOptions(ideOptions: IDEOptions, type: "browser" | "desktop")
 function renderIdeOption(option: IDEOption, selected: boolean, onSelect: () => void): JSX.Element {
     const card = <SelectableCard className="w-36 h-40" title={option.title} selected={selected} onClick={onSelect}>
         <div className="flex justify-center mt-3">
-            <img className="w-16 filter-grayscale self-center"
-                src={option.logo} alt="logo" />
+            <div className="w-16 h-16 text-center">
+                {option.logo && option.logo.length > 0 && <>
+                    <img className="w-16 filter-grayscale self-center"
+                        src={option.logo} alt="logo" />
+                </>}
+            </div>
```

Comment:
All `IDEOption` should always have a `logo`, right? If so, maybe you could undo the extra `option.logo` check here?

Reply:
yes